ui/src/app/shared/status/status.icon.html

```diff
@@ -6,5 +6,5 @@
     <i class="remove red icon" *ngSwitchCase="'Fail'"></i>
     <i class="ban grey icon" *ngSwitchCase="'Disabled'"></i>
     <i class="ban grey icon" *ngSwitchCase="'Skipped'"></i>
-    <i class="wait orange icon" *ngSwitchCase="'Waiting'"></i>
+    <i class="wait blue icon" *ngSwitchCase="'Waiting'"></i>
```

Comment:
Building is blue. Waiting have to be another color. We can update for another color, but update have to be done on UI2 & CLI (cds ui, then "m")